src/main/java/com/timgroup/statsd/NonBlockingStatsDClient.java

```diff
@@ -491,6 +491,44 @@ public void recordEvent(final Event event, final String... tags) {
                 title.length(), text.length(), title, text, eventMap(event), tagString(tags)));
     }
 
+    /**
+     * Records a run status for the specified named service check.
+     *
+     * <p>This method is a DataDog extension, and may not work with other servers.</p>
+     *
+     * <p>This method is non-blocking and is guaranteed not to throw an exception.</p>
+     *
+     * @param sc
+     *     the service check object
+     */
+    @Override public void recordServiceCheckRun(ServiceCheck sc) {
+        send(toStatsDString(sc));
+    }
+    
+    private String toStatsDString(ServiceCheck sc) {
+        StringBuilder sb = new StringBuilder();
+        sb.append(String.format("_sc|%s|%d", sc.getName(), sc.getStatus()));
```

Comment:
It would be nice if you could link here to the datadog wire protocol documentation.